src/webgl/p5.Camera.js

```diff
@@ -1600,13 +1600,12 @@ p5.Camera = class Camera {
     this.cameraMatrix = cam.cameraMatrix.copy();
     this.projMatrix = cam.projMatrix.copy();
 
-    // If the target camera is active, update uMVMatrix and uPMatrix.
     if (this._isActive()) {
-      this._renderer.uMVMatrix.mat4 = this.cameraMatrix.mat4.slice();
+      this._renderer.uModelMatrix.mat4 = this.modelMatrix.mat4.slice();
```

Comment:
This was something that I worked on in https://github.com/processing/p5.js/pull/6633#issuecomment-1859290684. We can do `this._renderer.uModelMatrix.set(this.modelMatrix);` instead of messing around with the "internals" of the matrix.

Reply:
yes, I updated these!